community/en/census/tf20_census.ipynb

```diff
@@ -0,0 +1,1422 @@
+        "                          steps_per_epoch=int(_NUM_EXAMPLES['train']/BATCH_SIZE), \n",
+        "                          validation_steps=int(_NUM_EXAMPLES['validation']/BATCH_SIZE), \n",
+        "                          epochs=EPOCHS, \n",
+        "                          callbacks=[lr_decay],\n",
+        "                          verbose=2)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Dm2VTyIUhG6y",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "**Reference:** Traditional Keras training mode.\n",
+        "\n",
+        "```keras_model.fit(x=train_x, y=train_y, epochs=40, validation_data=(test_x, test_y), verbose=1, callbacks=[lr_decay])```"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "fW7vTPm2pd2l",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Visualize Model history"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "lfuLSeSXucI4",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Visualize History for Accuracy.\n",
+        "plt.title('Keras Model accuracy')\n",
+        "plt.ylabel('accuracy')\n",
+        "plt.xlabel('epochs')\n",
+        "plt.plot(history.history['acc'])\n",
+        "plt.plot(history.history['val_acc'])\n",
+        "plt.legend(['training', 'testing'], loc='upper left')\n",
+        "plt.show()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "yU1TzdlY0i-Y",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Visualize History for Loss.\n",
+        "plt.title('Keras Model loss')\n",
+        "plt.plot(history.history['loss'])\n",
+        "plt.plot(history.history['val_loss'])\n",
+        "plt.ylabel('loss')\n",
+        "plt.xlabel('epoch')\n",
+        "plt.legend(['training', 'testing'], loc='upper left')\n",
+        "plt.show()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "r40PI5eOHXtd",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Part 2. Serving a Keras model"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "gmTiOQObmTqd",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Export model for Serving"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "d4BaDUJzmTW_",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Exporting model to GCS. (Optional)\n",
+        "export_path = tf.keras.experimental.export(keras_model, os.path.join('gs://', BUCKET_NAME, 'keras_export'))\n",
```

Comment:
The notebook ran fine up to this point. Can you open access to the storage bucket?

```
PermissionDeniedError: Error executing an HTTP request: HTTP response code 403 with body '{
 "error": {
  "errors": [
   {
    "domain": "global",
    "reason": "forbidden",
    "message": "{me} does not have storage.objects.get access to tony-dev/keras_export/1550680693."
   }
  ],
  "code": 403,
  "message": "{me} does not have storage.objects.get access to tony-dev/keras_export/1550680693."
 }
}
'
	 when reading metadata of gs://tony-dev/keras_export/1550680693
```